types/artifacts.d.ts

```diff
@@ -12,177 +14,177 @@ import speedline from 'speedline-core';
 import TextSourceMap from '../core/lib/cdt/generated/SourceMap.js';
 import {ArbitraryEqualityMap} from '../core/lib/arbitrary-equality-map.js';
 import type { TaskNode as _TaskNode } from '../core/lib/tracehouse/main-thread-tasks.js';
-import AuditDetails from './lhr/audit-details'
-import Config from './config';
-import Gatherer from './gatherer';
-import {IcuMessage} from './lhr/i18n';
-import LHResult from './lhr/lhr'
-import Protocol from './protocol';
+import AuditDetails from './lhr/audit-details.js'
+import Config from './config.js';
+import Gatherer from './gatherer.js';
+import {IcuMessage} from './lhr/i18n.js';
+import LHResult from './lhr/lhr.js'
+import Protocol from './protocol.js';
 import Util from './utility-types.js';
 import Audit from './audit.js';
 
-export interface Artifacts extends BaseArtifacts, GathererArtifacts {}
+interface Artifacts extends Artifacts.BaseArtifacts, Artifacts.GathererArtifacts {}
 
-export type FRArtifacts = Util.StrictOmit<Artifacts,
-  | 'Fonts'
-  | 'Manifest'
-  | 'MixedContent'
-  | keyof FRBaseArtifacts
->;
+declare module Artifacts {
```

Comment:
What is the restructuring in here for? It's a lot of churn but it doesn't seem like anything fundamentally changes (just `import Artifacts` vs `import {Artifacts}`)

Reply:
This was also from an earlier version, I'll try without this update